topk-py/src/query/query.rs

```diff
@@ -105,7 +105,7 @@ impl Query {
     }
 
     #[pyo3(signature = (expr, k, asc=false))]
-    pub fn top_k(&self, expr: LogicalExpr, k: u64, asc: bool) -> PyResult<Self> {
+    pub fn topk(&self, expr: LogicalExpr, k: u64, asc: bool) -> PyResult<Self> {
```

Comment:
nit: let's add a (deprecated) alias for now, so existing use cases still work, we can drop it later.